.github/workflows/test-suite.yml

```diff
@@ -7,13 +7,18 @@ on:
   pull_request:
     branches: ["master"]
 
+defaults:
+  run:
+    shell: bash
+
 jobs:
   tests:
-    name: "Python ${{ matrix.python-version }}"
-    runs-on: "ubuntu-latest"
+    name: "Python ${{ matrix.python-version }} (${{ matrix.os }})"
+    runs-on: "${{ matrix.os }}-latest"
 
     strategy:
       matrix:
+        os: ["ubuntu", "windows"]
```

Comment:
I'm not sure - I'm pleased to see us fix the test cases up for supporting windows, tho do we want to double the time our test runs take to complete?

Reply:
I agree with keeping it simple, and not add windows.